packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -204,11 +204,18 @@ abstract class FlutterDriver {
     await sendCommand(Tap(finder, timeout: timeout));
   }
 
-  /// Waits until [finder] locates the target.
+  /// Waits until [finder] locates the target. [finder] will wait until
```

Comment:
Sentences in docs should be grammatically correct, i.e. start with a capital letter and don't have random capital letters in the middle of it.

Reply:
fixed